my-app/public/index.html

```diff
@@ -1,122 +1,36 @@
-<!doctype html>
```

Comment:
Note that this change will lose the homepage template that we initially had, and replace it with essentially a header and a blank page. Should I work on getting the old homepage back into the dashboard or frontpage branch, or should we just worry about that later on?

Also the header on the old homepage is different than the header I've been working on, so we would have to come together and see which one we want to pick.

Reply:
Good point. I'll work on the frontend branch to integrate homepage and other aspects to this pull request.

Reply:
Okay sounds good! You should be able to post all of the homepage body html into the "landing" component and try to get that to work. Leave out the header though because that's a separate component.

Reply:
Sounds good. I'll give it a shot!